### PR TITLE
Edgelet build fix for Mac OS

### PR DESCRIPTION
**Pull request by massand**
A quick PR for fixing MacOS edgelet build. The last commit on that file did some refactor that ignored non-Windows, non-Linux. If we don't want zero uptime reporting, we could use libc I guess.

### CODE REVIEW

**Comment by arsing** [comment]
>If we don't want zero uptime reporting, we could use libc I guess.

No, it's fine. It's a niche request so it's not worth the effort.

File: edgelet/edgelet-docker/src/runtime.rs [hunk 1]
```diff
@@ -631,6 +631,9 @@ impl ModuleRuntime for DockerModuleRuntime {
                 })
             });
 
+        #[cfg(not(any(windows, target_os = "linux")))]
+        let uptime: u64 = 0;
+
         #[cfg(target_os = "linux")]
```
Comment (dmolokanov) [inline edgelet/edgelet-docker/src/runtime.rs:637]: will this work if you simply change to this?
```suggestion
        #[cfg(target_os = "unix")]
```

Comment (massand) [reply edgelet/edgelet-docker/src/runtime.rs:635]: target_os = "unix" doesn't build. There's actually a "macos" target_os. But, libc doesn't have sysinfo for "macos" platform.

Comment (dmolokanov) [reply edgelet/edgelet-docker/src/runtime.rs:635]: `#[cfg(unix)]` its not a target_os, but family. but it won't work anyways due to sysinfo limitations...

**Comment by dmolokanov** [comment]
You don't like Kodiak?

**Pull request merged**